pos_print_receipt_backend/models/pos_order.py

```diff
@@ -0,0 +1,63 @@
+from odoo import _, api, fields, models
+
+
+class PosOrder(models.Model):
+    _inherit = "pos.order"
+
+    receipt_sent = fields.Boolean(
+        readonly=True,
+        default=False,
+        copy=False,
+        help="It indicates that the receipt has been sent.",
+    )
+
+    @api.multi
+    def receipt_print(self):
+        self.filtered(lambda o: not o.receipt_sent).write(
+            {"receipt_sent": True}
+        )
+        return (
+            self.env.ref("pos_print_receipt_backend.action_report_receipt")
+            .with_context(discard_logo_check=True)
+            .report_action(self)
+        )
+
+    @api.multi
+    def action_receipt_sent(self):
+        """ Open a window to compose an email, with
+            email_template_pos_order_receipt message loaded by default
+        """
+        self.ensure_one()
+        template = self.env.ref(
+            "pos_print_receipt_backend.email_template_pos_order_receipt", False
+        )
+        compose_form = self.env.ref(
+            "pos_print_receipt_backend.pos_order_send_wizard_form", False
+        )
+        # have model_description in template language
+        lang = self.env.context.get("lang")
```

Comment:
In case of sending an email. I think it should be in the lang of the customer (res_partner).  From what I remember it's configured at the email template level.